### ISSUE TITLE
Allow this plugin to work with chai version 2+

### ISSUE THREAD

**Issue by kayhadrin** [gaslight/chai-datetime#20]
After reviewing the [release notes](https://github.com/chaijs/chai/blob/master/ReleaseNotes.md) of Chai, I think that there wouldn't be any harm to allow chai-datetime to use `>1.9.0` instead of `^1.9.0`.

I'm using it already manually with chai 2.1.1 but I want to avoid getting warning messages due to npm package version issues.

```
npm WARN unmet dependency ...\node_modules\chai-datetime requires chai@ but will load
npm WARN unmet dependency ...\node_modules\chai,
npm WARN unmet dependency which is version 2.1.1
project@0.0.1 project
├── chai@2.1.1 invalid
├── chai-as-promised@4.3.0
├── chai-datetime@1.3.1
...
npm ERR! invalid: chai@2.1.1 c:\itrac\workspace\itrac\itrac-web\node_modules\chai
npm ERR! not ok code 0
```

**Comment by mguterl** [comment]
@kayhadrin I need to understand a bit more about semver before I'm comfortable committing this change. I think I would feel more comfortable if it was a bit more restrictive, until it's been tested on future major versions. I'll think about this some more today and if you have any insight that would be appreciated.

**Comment by kayhadrin** [comment]
It perfectly makes sense to err on the side of caution regarding the management of dependency versions.

As you probably know, in an application using semver, the fact that the major version number is incremented means that some breaking changes were introduced.
So it's understandably daunting to let your own code allow `chai@>1.9.0`  by default.

Now though, I've reviewed the chai change logs and saw that it introduced a `addChainableNoop` function - which the chai-datetime plugin does not use. 
Since that brought some unforeseen negative effects on some other plugins, they reverted it in version 2.0.0 - by removing `addChainableNoop`.
The rest of their updates do not include substantial changes regarding this plugin again.
But in my experience, your plugin works fine with `chai@2.1.1`.

If I was in your situation:

- When you limit it with `chai@^1.9.0`, it means that it's up to you to review the compatibility of new major versions of dependencies. 
People like me would come sometimes to request updates to this version number to allow for new updates.

- When you set `>1.9.0`, which is admittedly more optimistic, you let your plugin be used with any sort of major updates from your dependencies. 
People like me would come sometimes to advise you that incompatibilities and issues have aroused when the time comes.

I'm just glad you decided to share this useful plugin, so I'll be happy with your final decision.
The choice is yours. :-)

**Comment by mguterl** [comment]
Fixed in #21. I'll be creating a new release in the next few days.

**Comment by kayhadrin** [comment]
Thanks :-)
